apps/condo/domains/ticket/utils/handlers.spec.js

```diff
@@ -42,30 +42,6 @@ describe('Ticket request event detection', () => {
         )
     })
 
-    it('correctly detects same person as assignee and executor connection on ticket create', () => {
```

Comment:
why do you want to delete tests??

Reply:
These tests were case when executor is the same with assignee in ticket, we discussed this logic with Mikhail Rumanovsky, and decided to remove it, so those tests are not needed any more.

Reply:
I don’t understand. What the code should do in such case?

Reply:
We decided to remove that logic so these tests are not needed naymore.